src/app/zap-templates/zcl/data-model/chip/general-commissioning-cluster.xml

```diff
@@ -83,5 +116,15 @@ limitations under the License.
       <arg name="ErrorCode" type="CommissioningErrorEnum"/>
       <arg name="DebugText" type="char_string"/>
     </command>
+    <command source="client" code="0x06" name="SetTCAcknowledgements" optional="true" response="SetTCAcknowledgementsResponse" cli="chip fabric_commissioning settcacknowledgements">
+      <description>This command sets the user acknowledgements received in the Enhanced Setup Flow Terms and Conditions into the node.</description>
+      <arg name="TCVersion" type="int16u" min="0x0000" max="0xFFFF"/>
+      <arg name="TCUserResponse" type="bitmap16" min="0x0000" max="0xFFFF"/>
```

Comment:
Why bother with min/max if it's just full-range?

Reply:
Removed all `min="0x0000" max="0xFFFF"`